app.py

```diff
@@ -6,7 +6,21 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    tasks = session.get_sorted_items()
+    return render_template('index.html', tasks=tasks)
+
+@app.route('/task', methods=['POST'])
+def addTask():
+    title = request.form.get('title')
+    session.add_item(title)
+    return index()
```

Comment:
You'll want to redirect back to the homepage here rather than serve the template directly (to ensure the correct url is displayed)